File: courses/examples/bmi.py
# Body Mass Index calculator

print("Mass (kg): ")
mass = int(input())

print("Height (cm): ")
height = int(input())

# předod výšky na metry
height = height / 100.0

# výpočet (bez jakýchkoli kontrol)
bmi = mass / (height * height)

# výpis výsledku
print("BMI = ", bmi)

File: courses/examples/bmi.c
#include <stdio.h>

int main(void)
{
    float mass, height;
    float bmi;

    printf("Mass (kg): ");
    scanf("%f", &mass);

    printf("Height (cm): ");
    scanf("%f", &height);

    height = height / 100;

    bmi = mass / (height * height);
    printf("BMI = %10.2f\n", bmi);

    return 0;
}

